package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "ferdi",
-  "productName": "Ferdi",
-  "desktopName": "ferdi.desktop",
-  "appId": "com.kytwb.ferdi",
+  "name": "ferdium",
+  "productName": "Ferdium",
+  "desktopName": "ferdium.desktop",
+  "appId": "com.kytwb.ferdium",
```

Comment:
The appId should be a reverse domain name of something controlled by Ferdium (like `org.ferdium.ferdium-app` or `com.github.ferdium.ferdium-app`). [Relevant Apple Doc](https://help.apple.com/xcode/mac/current/#/dev618af4e67)